cachi2/core/package_managers/generic/main.py

```diff
@@ -102,3 +103,29 @@ def _load_lockfile(lockfile_path: RootedPath, output_dir: RootedPath) -> Generic
                 ),
             )
     return lockfile
+
+
+def _generate_sbom_components(lockfile: GenericLockfileV1) -> list[Component]:
+    """Generate a list of SBOM components for a given lockfile."""
+    components: list[Component] = []
+
+    for artifact in lockfile.artifacts:
+        name = Path(artifact.target).name
+        url = str(artifact.download_url)
+        component = Component(
+            name=name,
+            purl=PackageURL(
+                type="generic",
+                name=name,
+                qualifiers={
+                    "download_url": url,
+                    "checksums": ",".join([f"{a}:{h}" for a, h in artifact.checksums.items()]),
+                },
+            ).to_string(),
+            package_manager="generic",
```

Comment:
Is `package_manager` an attribute in the `Component` class ?

Reply:
Good catch, that's a leftover from something I was testing. Removed.